frameworks/g-cloud-9/manifests/search_filters.yml

```diff
@@ -5,3 +5,18 @@
     - serviceCategoriesSoftware
     - serviceCategoriesHosting
     - serviceCategoriesSupport
+-
+  name: User support
+  slug: user-support
+  questions:
+    - emailSupport
+    - phoneSupport
+    - webChatSupport
+-
+  name: Government security clearances
+  questions:
```

Comment:
If slug is needed for user support, it should probably be here for security clearances and security certification.